command/state_mv.go

```diff
@@ -223,7 +220,7 @@ func (c *StateMvCommand) Run(args []string) int {
 
 				// Update the address before adding it to the state.
 				rs.Addr = addrTo
-				stateTo.Module(addrTo.Module).Resources[addrTo.Resource.String()] = rs
+				stateTo.EnsureModule(addrTo.Module).Resources[addrTo.Resource.String()] = rs
```

Comment:
I like that this also clears up a bit of redundancy (more like unnecessary code) since EnsureModule is safe to run in place of Module (here, that is)!